pages/edit/edit.css

```diff
@@ -0,0 +1,120 @@
+* {
+    box-sizing: border-box
+}
+
+body {
+  text-align: center;
+}
+
+.logo-container img {
+  width: 15%;
+}
+
+form {
+  width: 40%;
+  max-width: 520px;
+  margin: auto;
+  border: 1px solid grey;
+  border-radius: 10px;
+  overflow: hidden;
+}
+
+.container {
+  padding-bottom: 16px;
+}
+
+form label {
+  font-weight: bold;
+}
+
+input {
+  width: 100%;
+  padding: 15px;
+  margin: 5px 0 5px 0;
+  display: flex;
+  border: none;
+  background: #f1f1f1;
+  border-radius: 0.5rem;
+}
+
+input:focus {
+  background-color: #ddd;
+  outline: none; 
+}
+
+hr {
+  border: 1px solid #f1f1f1;
+  margin-bottom: 25px;
+}
+
+.registerbtn {
```

Comment:
maybe better to name variables in general style `register-btn`

Reply:
Yes, I changed naming of register-btn, login-btn, edit-btn, delete-btn in Html and css files.